golem_messages/constants.py

```diff
@@ -44,3 +45,8 @@
         message.concents.ForceReportComputedTask: (2 * MMTT + MAT),
     },
 )
+
+
+class MarketType(str, Enum):
```

Comment:
If this is only used in `TaskHeader` than I believe it should be an inner class of it.

Reply:
Actually it would be great if one could access those values from Golem itself.

Reply:
`TaskHeader.MARKET_TYPE`?

Reply:
Why bind this type to a particular message class?

Reply:
> If this is only used in `TaskHeader` than I believe it should be an inner class of it.

If it's stored in some other place it might be introduced as a stand alone constant. But as far as I can see it's only used in `TaskHeader`.

Reply:
OK